code/datums/outfits/outfit_admin.dm

```diff
@@ -219,25 +219,41 @@
 	// Will show as ? on sec huds, as this is not a recognized rank.
 
 /datum/outfit/admin/nt_undercover
-	name = "NT Undercover Operative"
-	// Disguised NT special forces, sent to quietly eliminate or keep tabs on people in high positions (e.g: captain)
+	name = "NT Undercover Agent"
+	var/access_level = "Emergency Response Team Leader"
+	// Ghost role for undercover antics on behalf of NT. Much weaker than the "NT Undercover Officer" below.
+	//Lacks CC implant and CC access. Default weapon is silenced saber SMG.
 
 	uniform = /obj/item/clothing/under/color/black
```

Comment:
Maybe use under/color/random instead of under/color/black
Otherwise, if an admin spawns 5 of these, they'll all appear in the exact same black outfit, which makes them super-easy to identify once you know what to look for.